scripts/doc-checker.py

```diff
@@ -13,16 +13,109 @@
 from typing import List
 import sys
 
-sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), "..")))
+MODIN_PATH = os.path.abspath(os.path.join(os.path.dirname(__file__), ".."))
+sys.path.insert(0, MODIN_PATH)
 
 # error codes that pandas test in CI
 # https://numpydoc.readthedocs.io/en/latest/validation.html#built-in-validation-checks
 NUMPYDOC_BASE_ERROR_CODES = {
-    *("GL01", "GL02", "GL03", "GL05", "GL06", "GL07", "GL09", "GL10"),
-    *("SS02", "SS04", "SS05", "PR03", "PR04", "PR05", "PR10", "RT01"),
-    *("RT04", "RT05", "SA02", "SA03"),
+    *("GL01", "GL02", "GL03", "GL05", "GL06", "GL07", "GL08", "GL09", "GL10"),
+    *("SS02", "SS03", "SS04", "SS05", "PR01", "PR02", "PR03", "PR04", "PR05"),
+    *("PR10", "RT01", "RT04", "RT05", "SA02", "SA03"),
 }
 
+""" TEST CUSTOM CHECK
```

Comment:
what's this? 😄

Reply:
>> @anmyachev please give a thought at implementing a separate check for optional, default: foo part of parameter description (or whatever form we agreed upon) or see if numpydoc already has such a check.

There is no such built-in check, but it's easy enough to add your own check and this code demonstrates it. It remains to choose what we want.

My favorite option is where we always write for optional arguments like this: `default [SOME VALUE]`

Reply:
I would prefer `default: VALUE` (i.e. add a colon for more clear delimiting).

My original comment was related to you adding a giant comment with code inside :) Why didn't you just add it as a real code? This is still the check we want to have.

Speaking of additional checks - we should add checks for "modin" vs "Modin", "numpy" vs "NumPy", etc. (ref: https://github.com/modin-project/modin/issues/2934#issuecomment-815494134), but note that this is for standalone words, not for types like `modin.pandas.DataFrame` or `pandas.Series`.

Reply:
Added check on `default: VALUE`